src/daemon/plugin.h

```diff
@@ -43,13 +43,11 @@
 #define DS_TYPE_ABSOLUTE 3
 
 #define DS_TYPE_TO_STRING(t)                                                   \
-  (t == DS_TYPE_COUNTER)                                                       \
-      ? "counter"                                                              \
-      : (t == DS_TYPE_GAUGE)                                                   \
-            ? "gauge"                                                          \
-            : (t == DS_TYPE_DERIVE)                                            \
-                  ? "derive"                                                   \
-                  : (t == DS_TYPE_ABSOLUTE) ? "absolute" : "unknown"
+  (t == DS_TYPE_COUNTER)    ? "counter"                                        \
+  : (t == DS_TYPE_GAUGE)    ? "gauge"                                          \
+  : (t == DS_TYPE_DERIVE)   ? "derive"                                         \
+  : (t == DS_TYPE_ABSOLUTE) ? "absolute"                                       \
+                            : "unknown"
```

Comment:
Unrelated code formatting change.  Drop it (it's something that changes in v6 any way).

Reply:
Sorry, I ran format.sh over all files. Dropped.

Reply:
Cr*p, I had not realized clang-format rules had changed/updated since this file had last been updated.

In that case it's better to add back.  Sorry about that! :-/

Reply:
OK, no problem! Undone.